packages/hooks/use-draggable/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# @nextui-org/use-disclosure
```

Comment:
CHANGELOG is generated.

Reply:
> CHANGELOG is generated.

I don't know how to generate CHANGELOG, can you help me?

Reply:
It's generated from changeset. You don't need to do it on your own. You just need to create [one](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md).

Reply:
@wingkwong Thanks❤️! I have finish all. Review pls.